packages/iTwinUI-react/src/core/DatePicker/DatePicker.test.tsx

```diff
@@ -307,3 +307,102 @@ it('should navigate between years', () => {
   date.setMonth(date.getFullYear() + 1);
   assertMonthYear(container, 'February', '2019');
 });
+
+it('should return selected date range', () => {
+  const onClick = jest.fn();
+  const { container, getByText } = render(
+    <DatePicker
+      startDate={new Date(2020, 5, 5)}
+      endDate={new Date(2020, 5, 10)}
+      onChange={onClick}
```

Comment:
Check that it being called with correct values in all your tests.

Reply:
Good suggestion! The unit tests are now fixed

Reply:
So are you going to check with what `onClick` was called? 😁

Reply:
`onClick` checks have been added to the unit test